src/commands/dev/index.js

```diff
@@ -102,16 +102,15 @@ const getAddonsUrlsAndAddEnvVariablesToProcessEnv = async ({ api, site, flags })
   }
 }
 
-const addDotFileEnvs = async ({ site }) => {
+const addDotFileEnvs = async ({ site, log }) => {
+  // add .env file environment variables
   const envSettings = await getEnvSettings(site.root)
-  if (envSettings.file) {
-    console.log(
-      `${NETLIFYDEVLOG} Overriding the following env variables with ${chalk.blue(
-        path.relative(site.root, envSettings.file)
```

Comment:
Changed the function to return just the filenames, so we won't have to trim the full path